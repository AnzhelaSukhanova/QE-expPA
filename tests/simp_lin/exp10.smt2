(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 4))
(declare-fun z () (_ BitVec 4))
(assert
 (exists ((x (_ BitVec 4))) (and 
 (or (bvult x (bvadd (bvmul (bvshl #b0001 y) #b0101) #b0111)) (= x (bvadd (bvmul (bvshl #b0001 y) #b0101) #b0111))) 
 (or (bvult (bvmul (bvadd (bvshl #b0001 y) z) #b1000) x) (= (bvmul (bvadd (bvshl #b0001 y) z) #b1000) x)))))
