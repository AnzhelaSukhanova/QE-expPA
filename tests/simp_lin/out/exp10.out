(not (bvult (bvadd (bvshl #x5 y) #x7) (bvmul #x8 (bvadd z (bvshl #x1 y)))))
